Anguandia/quickcredit
# Integrate database into get loans functionality

#### Purpose of this pull request

Integrate database into get loans functionality

#### Tasks to be accomplished

- Add query to retrieve all loans from loans table in database

#### How to manually test this feature

- Clone this repo and run npm install to install this app locally
- Start the app with either npm start or debug=quickcredit npm run dev
commands in your terminal
- Proceed to create a loan with a POST/api/v1/loans request supplying a
json object with values for the fields of email(user), amount(float) and
tenor(integer)
- Create as any other loans as you please
- Restart the application and run a GET/api/v1/loans request on postman
using localhost and port 3000
- You should see all the previously created loans in the response

[finishes]
